src/site/_filters/paginate.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Take array of paginated hrefs and select the pages to display.
+ * @param {string[]} pages An array of hrefs of all the pages.
+ * @param {string} current Href of current page.
+ * @return {object[]} An array of up to 10 items to display, including href and index.
+ */
+module.exports = function paginate(pages, current) {
+  const currentIndex = pages.findIndex((page) => page === current);
```

Comment:
Since page is a string, maybe consider indexOf instead of findIndex for less code? There's no need for a callback.